src/main.py

```diff
@@ -458,6 +608,32 @@ async def knative_request(
             )
 
 
+async def redis_stream_request(
+    redis_client: redis.Redis,
+    redis_stream: str,
+    body: dict[str, typing.Any],
+) -> None:
+    """Makes redis stream request.
+
+    Parameters
+    ----------
+    in_process_requests_gauge : `prometheus_client.Gauge`
+        A gauge to be updated with the start and end of the request.
+    client : `redis.Redis`
+        The redis stream client.
+    redis_stream : `string`
```

Comment:
Links to `string` are still invalid:
```suggestion
    redis_stream : `str`
```